src/sagemaker/session.py

```diff
@@ -450,6 +450,9 @@ def tune(
         early_stopping_type="Off",
         encrypt_inter_container_traffic=False,
         vpc_config=None,
+        train_use_spot_instances=False,
```

Comment:
Should we allow the user to provide a train_max_wait similar to the Estimator call?

https://github.com/aws/sagemaker-python-sdk/blob/b33d1a29b77ec86d0c106ce32f97053652ff7933/src/sagemaker/estimator.py#L86

Reply:
We don't need to pass `train_max_wait` to HyperparameterTuner. It has already been passed as `stop_condition` from the estimator of a HyperparameterTuner instance.

https://github.com/aws/sagemaker-python-sdk/blob/b33d1a29b77ec86d0c106ce32f97053652ff7933/src/sagemaker/job.py#L85

This is called from here:
https://github.com/aws/sagemaker-python-sdk/blob/b33d1a29b77ec86d0c106ce32f97053652ff7933/src/sagemaker/tuner.py#L853